src/examples/TouchableWithoutFeedbackExamplePage.tsx

```diff
@@ -0,0 +1,117 @@
+'use strict';
+import {Text} from 'react-native';
+import React, {useState} from 'react';
+import {Example} from '../components/Example';
+import {Page} from '../components/Page';
+import {useTheme} from '@react-navigation/native';
+import {TouchableWithoutFeedback} from 'react-native-windows';
+import {Banner} from '@react-native-material/core';
+
+export const TouchableWithoutFeedbackExamplePage: React.FunctionComponent<{}> = () => {
+  const [title, setTitle] = useState(0);
+  const [focus, setFocus] = useState(false);
+  const {colors} = useTheme();
+
+  const example1jsx = `<TouchableWithoutFeedback
+    >
+    <Text style={{color: colors.text}}>TouchableWithoutFeedback</Text>
+  </TouchableWithoutFeedback>`;
+
+  const example2jsx = `<TouchableWithoutFeedback
```

Comment:
nit: you'll want the formatting of these jsx snippets to match the formatting of the jsx itself. This way all of our code snippets are formatted the same.

 ">" tags should not be on their own line and make sure the tabbing is correct (i.e. only the child element should be indented inwards)